tests/bluetooth/ctrl_isoal/testcase.yaml

```diff
@@ -1,44 +1,82 @@
+      - rx_unframed_dbl_pdu
+      - rx_unframed_dbl_split
+      - rx_unframed_multi_split
+      - rx_unframed_multi_split_on_border
+      - rx_unframed_long_pdu_short_sdu
+      - rx_unframed_dbl_pdu_prem
+      - rx_unframed_single_pdu_err
+      - rx_unframed_seq_err
+      - rx_unframed_seq_pdu_err1
+      - rx_unframed_seq_pdu_err2
+      - rx_unframed_padding
+      - rx_unframed_padding_no_end
+      - rx_unframed_padding_error1
+      - rx_unframed_padding_error2
+      - rx_unframed_padding_error3
+      - rx_unframed_zero_len_packet
+      - rx_unframed_seq_err_zero_length
+      - rx_unframed_dbl_pdu_no_end
+      - rx_unframed_dbl_pdu_invalid_llid1
+      - rx_unframed_dbl_pdu_invalid_llid2
+      - rx_unframed_dbl_pdu_invalid_llid2_pdu_err
+      - rx_framed_single_pdu_single_sdu
+      - rx_framed_trppl_pdu_single_sdu
+      - rx_framed_trppl_pdu_dbl_sdu
+      - rx_framed_zero_length_sdu
+      - rx_framed_dbl_pdu_dbl_sdu_padding
+      - rx_framed_dbl_pdu_dbl_sdu_pdu_err1
+      - rx_framed_dbl_pdu_dbl_sdu_pdu_err2
+      - rx_framed_dbl_pdu_dbl_sdu_pdu_err3
+      - rx_framed_dbl_pdu_dbl_sdu_seq_err1
+      - rx_framed_trppl_pdu_single_sdu_pdu_err1
+      - rx_framed_trppl_pdu_single_sdu_pdu_err2
+      - rx_framed_trppl_pdu_single_sdu_pdu_err3
+      - rx_framed_trppl_pdu_single_sdu_seq_err1
+      - rx_framed_trppl_pdu_single_sdu_pdu_seq_err1
+      - rx_framed_trppl_pdu_dbl_sdu_pdu_err1
+      - rx_framed_trppl_pdu_dbl_sdu_pdu_err2
+      - rx_framed_trppl_pdu_dbl_sdu_pdu_err3
+      - rx_framed_trppl_pdu_dbl_sdu_seq_err1
+      - rx_framed_trppl_pdu_dbl_sdu_pdu_seq_err1
+      - rx_framed_single_invalid_pdu_single_sdu
+      - rx_framed_single_invalid_pdu_single_sdu_hdr_err
+      - rx_framed_trppl_pdu_dbl_sdu_seg_err1
+      - rx_framed_trppl_pdu_dbl_sdu_seg_err2
+      - source_isoal_test_create_destroy
+      - source_isoal_test_create_err
+      - source_invalid_ref
+      - tx_unframed_1_sdu_1_frag_1_pdu_maxPDU
+      - tx_unframed_1_sdu_1_frag_1_pdu_bufSize
+      - tx_unframed_1_sdu_1_frag_3_pdu
+      - tx_unframed_1_sdu_3_frag_1_pdu
+      - tx_unframed_1_sdu_3_frag_2_pdu
+      - tx_unframed_2_sdu_3_frag_4_pdu
+      - tx_unframed_2_sdu_3_frag_4_pdu_padding
+      - tx_unframed_1_zero_sdu_1_frag_1_pdu_maxPDU_padding
+      - tx_unframed_1_sdu_1_frag_pdu_alloc_err
+      - tx_unframed_1_sdu_1_frag_pdu_emit_err
+      - tx_framed_1_sdu_1_frag_1_pdu_maxPDU
+      - tx_framed_1_sdu_1_frag_1_pdu_bufSize
+      - tx_framed_1_sdu_1_frag_3_pdu
+      - tx_framed_1_sdu_3_frag_1_pdu
+      - tx_framed_1_sdu_3_frag_2_pdu
+      - tx_framed_2_sdu_3_frag_4_pdu
+      - tx_framed_2_sdu_3_frag_4_pdu_padding
+      - tx_framed_2_sdu_1_frag_2_pdu_refPoint2
+      - tx_framed_1_sdu_1_frag_1_pdu_refPoint3
+      - tx_framed_1_zero_sdu_1_frag_1_pdu_maxPDU
+      - tx_framed_1_sdu_1_frag_pdu_alloc_err
+      - tx_framed_1_sdu_1_frag_pdu_emit_err
+      - tx_framed_2_sdu_1_frag_pdu_timeout
+      - tx_framed_cis_fra_per_bv07c-
```

Comment:
Why is there a "-" at the end? Doesn't this exclude the test case?

Reply:
Good question! I didn't really consider it as I followed the existing format. 
But Google tells me that it's part of the YAML markup language denoting members of a list. In this case `testcases`.

Reply:
The trailing "-"? I don't think so

Reply:
Nice catch. Didn't notice.

Reply:
Removed.